lib/data/repositories/articles_directory_repository.dart

```diff
@@ -5,6 +5,8 @@ import 'package:farmsmart_flutter/data/model/article_entity.dart';
 import 'package:farmsmart_flutter/data/model/stage_entity.dart';
 import 'package:flutter/foundation.dart';
 
+// LH this file is deprecated !!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

Comment:
Please add the deprecated annotation: `@Deprecated('message')`

Reply:
nice, didn't know how to do that. 👍